### PR TITLE
Format all the docs

### PR DESCRIPTION
**Pull request by theblobscp**
Gives every file a nice format
### Formatters used:
[Prettier for html/css/js](https://prettier.io/)
[Xml Formatter](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-xml)

### CODE REVIEW

File: docs/assets/css/discord-logo.svg [hunk 1]
```diff
@@ -1 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 245 240"><path d="M104.4 103.9c-5.7 0-10.2 5-10.2 11.1s4.6 11.1 10.2 11.1c5.7 0 10.2-5 10.2-11.1.1-6.1-4.5-11.1-10.2-11.1zM140.9 103.9c-5.7 0-10.2 5-10.2 11.1s4.6 11.1 10.2 11.1c5.7 0 10.2-5 10.2-11.1s-4.5-11.1-10.2-11.1z"/><path d="M189.5 20h-134C44.2 20 35 29.2 35 40.6v135.2c0 11.4 9.2 20.6 20.5 20.6h113.4l-5.3-18.5 12.8 11.9 12.1 11.2 21.5 19V40.6c0-11.4-9.2-20.6-20.5-20.6zm-38.6 130.6s-3.6-4.3-6.6-8.1c13.1-3.7 18.1-11.9 18.1-11.9-4.1 2.7-8 4.6-11.5 5.9-5 2.1-9.8 3.5-14.5 4.3-9.6 1.8-18.4 1.3-25.9-.1-5.7-1.1-10.6-2.7-14.7-4.3-2.3-.9-4.8-2-7.3-3.4-.3-.2-.6-.3-.9-.5-.2-.1-.3-.2-.4-.3-1.8-1-2.8-1.7-2.8-1.7s4.8 8 17.5 11.8c-3 3.8-6.7 8.3-6.7 8.3-22.1-.7-30.5-15.2-30.5-15.2 0-32.2 14.4-58.3 14.4-58.3 14.4-10.8 28.1-10.5 28.1-10.5l1 1.2c-18 5.2-26.3 13.1-26.3 13.1s2.2-1.2 5.9-2.9c10.7-4.7 19.2-6 22.7-6.3.6-.1 1.1-.2 1.7-.2 6.1-.8 13-1 20.2-.2 9.5 1.1 19.7 3.9 30.1 9.6 0 0-7.9-7.5-24.9-12.7l1.4-1.6s13.7-.3 28.1 10.5c0 0 14.4 26.1 14.4 58.3 0 0-8.5 14.5-30.6 15.2z"/></svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 245 240">
```
Comment (Panquesito7) [inline docs/assets/css/discord-logo.svg:1]: IMO, it's best to have SVG file content in one line.

Comment (theblobscp) [reply docs/assets/css/discord-logo.svg:1]: I cant really commit those changes, but I can try

File: docs/assets/js/script.js [hunk 2]
```diff
@@ -45,19 +47,17 @@ devicon.controller('IconListCtrl', function($scope, $http, $compile) {
...
-    angular.forEach(data, function(devicon, key) {
-
+    angular.forEach(data, function (devicon, key) {
       // New icon format
       var icon = {
         name: devicon.name,
         svg: devicon.versions.svg,
         font: devicon.versions.font,
-        main: ""
+        main: "",
```
Comment (Thomas-Boi) [inline docs/assets/js/script.js:56]: I prefer not having dangling `,` commas.
```suggestion
        main: ""
```

**Review by Thomas-Boi — CHANGES_REQUESTED** [review]
Overall, the changes are good. Just need to fix the dangling comma and we should be fine.

Also, can you test the website on your end to ensure everything still works? Some changes (like putting things on new lines for JS) can cause the script to interpret things differently.

File: docs/assets/css/discord-logo.svg [hunk 3]
```diff
@@ -1 +1,4 @@
...
+    <path d="M104.4 103.9c-5.7 0-10.2 5-10.2 11.1s4.6 11.1 10.2 11.1c5.7 0 10.2-5 10.2-11.1.1-6.1-4.5-11.1-10.2-11.1zM140.9 103.9c-5.7 0-10.2 5-10.2 11.1s4.6 11.1 10.2 11.1c5.7 0 10.2-5 10.2-11.1s-4.5-11.1-10.2-11.1z" />
+    <path d="M189.5 20h-134C44.2 20 35 29.2 35 40.6v135.2c0 11.4 9.2 20.6 20.5 20.6h113.4l-5.3-18.5 12.8 11.9 12.1 11.2 21.5 19V40.6c0-11.4-9.2-20.6-20.5-20.6zm-38.6 130.6s-3.6-4.3-6.6-8.1c13.1-3.7 18.1-11.9 18.1-11.9-4.1 2.7-8 4.6-11.5 5.9-5 2.1-9.8 3.5-14.5 4.3-9.6 1.8-18.4 1.3-25.9-.1-5.7-1.1-10.6-2.7-14.7-4.3-2.3-.9-4.8-2-7.3-3.4-.3-.2-.6-.3-.9-.5-.2-.1-.3-.2-.4-.3-1.8-1-2.8-1.7-2.8-1.7s4.8 8 17.5 11.8c-3 3.8-6.7 8.3-6.7 8.3-22.1-.7-30.5-15.2-30.5-15.2 0-32.2 14.4-58.3 14.4-58.3 14.4-10.8 28.1-10.5 28.1-10.5l1 1.2c-18 5.2-26.3 13.1-26.3 13.1s2.2-1.2 5.9-2.9c10.7-4.7 19.2-6 22.7-6.3.6-.1 1.1-.2 1.7-.2 6.1-.8 13-1 20.2-.2 9.5 1.1 19.7 3.9 30.1 9.6 0 0-7.9-7.5-24.9-12.7l1.4-1.6s13.7-.3 28.1 10.5c0 0 14.4 26.1 14.4 58.3 0 0-8.5 14.5-30.6 15.2z" />
+</svg>
```
Comment (Panquesito7) [inline docs/assets/css/discord-logo.svg:4]: Missing an endline.
```suggestion
</svg>

```

File: docs/browserconfig.xml [hunk 4]
```diff
@@ -2,11 +2,11 @@
...
-      <wide310x150logo src="/mstile-310x150.png"/>
+      <square70x70logo src="/mstile-70x70.png" />
+      <square150x150logo src="/mstile-150x150.png" />
+      <square310x310logo src="/mstile-310x310.png" />
+      <wide310x150logo src="/mstile-310x150.png" />
       <TileColor>#da532c</TileColor>
     </tile>
   </msapplication>
-</browserconfig>
+</browserconfig>
```
Comment (Panquesito7) [inline docs/browserconfig.xml:12]:
```suggestion
</browserconfig>

```

File: docs/logos/safari-pinned-tab.svg [hunk 5]
```diff
@@ -284,37 +280,37 @@ c3 -5 2 -12 -3 -15 -5 -3 -9 1 -9 9 0 17 3 19 12 6z"/>
...
--11 15 -27 -5z"/>
-</g>
-</svg>
+l-23 -19 19 23 c18 21 26 27 26 19z" />
+                <path d="M1599 191 c11 -7 11 -9 0 -14 -8 -3 -4 -4 8 -4 17 1 19 4 11 14 -6 7
+-16 13 -22 13 -6 0 -5 -4 3 -9z" />
+                <path d="M2709 863 c-12 -16 -12 -17 2 -6 9 7 20 9 24 5 5 -4 5 -1 1 6 -9 16
+-11 15 -27 -5z" />
+        </g>
+</svg>
```
Comment (Panquesito7) [inline docs/logos/safari-pinned-tab.svg:316]:
```suggestion
</svg>

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Comment by Thomas-Boi** [comment]
Overall everything looks good. However, before we merge this, can you host the page and see if it's still running properly? You can use VS Code [extension](https://marketplace.visualstudio.com/items?itemName=yuichinukiyama.vscode-preview-server) for a quick check. Just run the server, check that everything is still working, and post a few pictures here to prove it.

Once that's done, I'll merge your changes.

**Comment by theblobscp** [comment]
> Overall everything looks good. However, before we merge this, can you host the page and see if it's still running properly? You can use VS Code [extension](https://marketplace.visualstudio.com/items?itemName=yuichinukiyama.vscode-preview-server) for a quick check. Just run the server, check that everything is still working, and post a few pictures here to prove it.
> 
> Once that's done, I'll merge your changes.

Everything looks the same, do I need to send the images still?

**Comment by Panquesito7** [comment]
> do I need to send the images

It'd be great if you could send some. 🙂

**Comment by Thomas-Boi** [comment]
I agreed with @Panquesito7. We just need a few images to ensure everything is good. Sometimes optimization (depending on the settings) could have unintended consequences

**Comment by theblobscp** [comment]
> I agreed with @Panquesito7. We just need a few images to ensure everything is good. Sometimes optimization (depending on the settings) could have unintended consequences

I'll send some tomorrow, I'll be on my pc then

**Comment by Thomas-Boi** [comment]
Hi @theblobscp, are you still working on this PR? We just need an image of what the website looks like with all the changes. Most formatter do a good job but sometimes, like with JS semi-colons, they might mess with the code, thus changing how the code works.

**Comment by theblobscp** [comment]
I kinda forgot about it lol, anyway I'll actually send it

**Comment by Thomas-Boi** [comment]
I'll close this PR for now. You can reopen it when you are ready to work on it

**Pull request closed without merging**